test/context/directory/attackProtection.test.js

```diff
@@ -9,10 +9,13 @@ describe('#directory context attack-protection', () => {
   it('should replace keywords', async () => {
     const files = {
       'attack-protection': {
-        'breached-password-detection.json': '{"enabled": "@@BREACH_PASSWORD_ENABLED@@", "shields": [], "admin_notification_frequency": [], "method": "##BREACH_PASSWORD_PROT_METHOD##"}',
-        'brute-force-protection.json': '{"enabled": "@@BRUTE_FORCE_PROT_ENABLED@@", "shields": ["block", "user_notification"], "mode": "count_per_identifier_and_ip", "allowlist": [], "max_attempts": 10}',
-        'suspicious-ip-throttling.json': '{"enabled": true, "shields": ["block", "admin_notification"], "allowlist": ["127.0.0.1"], "stage": {"pre-login": {"max_attempts": 100, "rate": 864000}, "pre-user-registration": {"max_attempts": 50, "rate": 1200}}}'
-      }
+        'breached-password-detection.json':
```

Comment:
**Thought:** I wonder how this file got checked-in without formatting. Since it's just a single file, I'm ok with it being here.